language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/codeaction/RangeBasedPositionDetailsImpl.java

```diff
@@ -17,29 +17,29 @@
 
 import io.ballerina.compiler.api.symbols.TypeSymbol;
 import io.ballerina.compiler.syntax.tree.NonTerminalNode;
-import org.ballerinalang.langserver.commons.codeaction.spi.NodeBasedPositionDetails;
+import org.ballerinalang.langserver.commons.codeaction.spi.RangeBasedPositionDetails;
 
 import java.util.Optional;
 
 /**
- * This class holds position details for the node-based code actions.
+ * This class holds position details for the range based code actions.
  *
  * @since 2.0.0
  */
-public class NodeBasedPositionDetailsImpl implements NodeBasedPositionDetails {
+public class RangeBasedPositionDetailsImpl implements RangeBasedPositionDetails {
     private final NonTerminalNode matchedTopLevelNode;
     private final NonTerminalNode matchedStmtNode;
     private final NonTerminalNode matchedCodeActionNode;
     private final TypeSymbol matchedTopLevelTypeSymbol;
     private final NonTerminalNode matchedDocumentableNode;
     private final NonTerminalNode enclosingDocumentableNode;
 
-    private NodeBasedPositionDetailsImpl(NonTerminalNode matchedTopLevelNode,
-                                         NonTerminalNode matchedStmtNode,
-                                         NonTerminalNode matchedCodeActionNode,
-                                         TypeSymbol matchedTopLevelTypeSymbol,
-                                         NonTerminalNode matchedDocumentableNode,
-                                         NonTerminalNode enclosingDocumentableNode) {
+    private RangeBasedPositionDetailsImpl(NonTerminalNode matchedTopLevelNode,
+                                          NonTerminalNode matchedStmtNode,
```

Comment:
```suggestion
                                         NonTerminalNode matchedStmtNode,
```
Shall we remove these extra spaces?

Reply:
This is expected, and it is due to the small offset it gets when refactoring `Node` to `Range` in the class name.